### PR TITLE
Add metrics for event counts

### PR DESCRIPTION
**Pull request by rossf7**
Towards https://github.com/giantswarm/giantswarm/issues/13309

## Checklist

- [x] Update changelog in CHANGELOG.md.

### CODE REVIEW

File: pkg/helmclient/rollback.go [hunk 1]
```diff
@@ -22,6 +22,8 @@ func (c *Client) Rollback(ctx context.Context, namespace, releaseName string, re
 		return microerror.Mask(err)
 	}
 
+	eventCounter.WithLabelValues(eventName).Inc()
```
Comment (tomahawk28) [inline pkg/helmclient/rollback.go:25]: In this case, if we have a problem during `c.rollback` function call, the counter value would not increase. 
What do you think to call it with `defer` above the rollback line?

**Review by tomahawk28 — COMMENTED** [review]
Looks good, just one question.

File: pkg/helmclient/rollback.go (hunk 1 above)
Comment (rossf7) [reply pkg/helmclient/rollback.go]: Good point. I used an anonymous func so the timer includes counting the rollback. PTAL

Comment (tomahawk28) [reply pkg/helmclient/rollback.go]: Thanks, maybe not only rollback but install, update also need the same changes, too. :)

Comment (rossf7) [reply pkg/helmclient/rollback.go]: There are already operatorkit metrics for errors. But OK lets use the same approach for each counter. Done PTAL

**Review by tomahawk28 — APPROVED** [review]
LGTM! Thanks for the change.

**Pull request merged**